### DISCUSSION TITLE
Possibility to show blank issue type when issue forms are enabled

### DISCUSSION THREAD

**Discussion by aliok** [community/community#101943] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

When I enable an [issue form](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository) in a repository, I see I have the option to leave "blank issues" enabled using the [`blank_issues_enabled` option](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) in `.github/ISSUE_TEMPLATE/config.yml` file.

This results in the new issue page showing the issue templates with big buttons, which is also great.

For example:

<img width="1266" alt="Screenshot 2024-01-25 at 09 40 51" src="https://github.com/community/community/assets/376732/a408ce14-c968-4168-9e87-736e3ffed738">

However, users seem to miss the "Don’t see your issue here? Open a blank issue." text below and get confused about how they can create blank issues.

I suggest the addition of a flag like `show_blank_issue_button` (which can default to false). In this case the issues page can show the buttons for templates and then a same-sized button for "Blank issue".

Issues page hypothetically can look like this then (manually edited HTML):

<img width="1266" alt="Screenshot 2024-01-25 at 09 46 10" src="https://github.com/community/community/assets/376732/63cc087a-84e8-4a3c-b18a-70eb7334f8c5">

As a workaround, we now add a contact link in `config.yml`, which is abusing the contact links:
```
contact_links:
  - name: Open a blank issue
    url: https://github.com/aliok/test-issue-templates/issues/new
    about: Please open a blank issue to if you don't see your issue type here.
```

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by shaunthom** [comment]
Hi man, You've got a good idea to make a "Blank issue" button that stands out next to the other template buttons. This would make it easier for people to see how to report different problems that don't match the templates. Until GitHub makes this change, your quick fix of adding a special link to open a blank issue is a smart move.

**Comment by linker34** [comment]
Hi man, You've got a good idea to make a "Blank issue" button that stands out next to the other template buttons. This would make it easier for people to see how to report different problems that don't match the templates . Until GitHub makes this change, your quick fix of adding a special link to open a blank issue is a smart move.
